python/dgl/graphbolt/dataloader.py

```diff
@@ -78,14 +78,21 @@ class MultiprocessingWrapper(dp.iter.IterDataPipe):
     num_workers : int, optional
         The number of worker processes. Default is 0, meaning that there
         will be no multiprocessing.
+    persistent_workers : bool, optional
+        If True, the data loader will not shut down the worker processes after a
+        dataset has been consumed once. This allows to maintain the workers
+        instances alive. If None, it will default to True when num_workers > 0.
     """
 
-    def __init__(self, datapipe, num_workers=0):
+    def __init__(self, datapipe, num_workers=0, persistent_workers=None):
```

Comment:
1. optional boolean is weird. 2. It is weird to make None as default True.

Just set the default to True, and `persistent_workers = (num_workers > 0) and persistent_workers` should be enough.